8.0.Dockerfile

```diff
@@ -29,6 +29,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=2
+ONBUILD ARG ODOO_WORKERS=2
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
```

Comment:
With these limits it's quite common that you cannot download a database backup. I'd increase both to 5 minutes minimum by default.

Reply:
IMO it's best to leave these at Odoo default and let a developer decide. I run my productions with these limits for security purposes. There is no reason a database backup should be able to complete from a production instance using user-facing methods